### ISSUE TITLE
UX - Keyboard accessibility

### ISSUE THREAD

**Issue by sean-eagles** [Canadian-Geospatial-Platform/geo.ca#162]
geoView users should be able to accomplish tasks using keyboard only.

Design - Vlad
Implementation - Jared
